src/app/SharpRaven/Data/SentryRequestFactory.cs

```diff
@@ -79,16 +94,16 @@ internal static bool HasCurrentHttpContextProperty
             get { return CurrentHttpContextProperty != null; }
         }
 
+        [JsonIgnore]
+        internal static bool CheckedForHttpContextProperty { get; set; }
```

Comment:
It was just moved and renamed (from 'CheckedForHttpContext') to make it more clear what it actually did.

But totally agree that it should be made private, but if you go that way there's a bunch of other things to look into as well... :smirk: